### ISSUE TITLE
bugs and unexpected behavior making this mod feel un-playable

### ISSUE THREAD

**Issue by LucidEye** [ElCeejo/animalia#64]
Minetest 5.7.0 running on Ubuntu MATE 22.04.3

Only been trying this mod for 2 days in Creative mode and have noticed some disappointing things that really make the mod feel unusable.

All tropical fish die immediately after being spawned.  When placed in any water, the fish immediately swim to the surface, jump out of the water, then come back down and sink to the bottom where they just sit, laying on their sides and flopping around until they eventually just disappear.

Livestock animals (horses, cows, sheep, pigs) do not fit through the fence gates???
I can lasso animals, no problem, but I can't lead them into any penned spaces I have built because they will not fit through the fence gates.  Only way to get them in is to tear down a block of fence and then lead them in and rebuild the fence.

Livestock animals like horses, cows, sheep, and pigs do not stay in fenced-in areas or pens,  They just disappear after a few minutes, sometimes while looking right at them... and even when they are lassoed to the fence... *POOF*.... just gone, completely vanished, along with the lasso as well.  Really frustrating to work at rounding up animals and they all just vanish without a trace while you are out rounding up more animals.

**Comment by PavanLuca** [comment]
Yeah, I sometimes saw animals disappear in front of me, too. I'd understand this behavior once the animals are far from the character of a certain distance but not this way, gives a horror feeling 😨. Ever heard or saw them die spontaneously? I often found meat pieces and other stuff when I returned to them, but there was no wolf or fox close there. Another horror situation. There's a killer somewhere following me 😱.

**Comment by DustyBagel** [comment]
The livestock not fitting though the fence gates may be a design choice because they don't look like they would fit through the gate, but I am not sure if that is the case. As a temporary fix, you could use a set of doors. The animals might be escaping because the minetest setting "tall fences and walls" isn't enabled. When this setting isn't enabled, fences and walls are treated like any other block but when the setting is enabled, the fence and wall blocks are slightly taller than one block so animals can't just jump over the fence or at least that is how I understood it.

**Comment by LucidEye** [comment]
> I often found meat pieces and other stuff when I returned to them, but there was no wolf or fox close there. Another horror situation. There's a killer somewhere following me 😱.

Except I am experiencing this behavior with damage turned OFF.  I'm just in play/build mode, so there are no game-generated predators or mobs that would be killing my livestock.

**Comment by PavanLuca** [comment]
I checked the mod settings and there are 2 options that can explain animals disappearing and finding their items on the ground, **Pest Mobs Spawn Chance** and **Predator Mobs Spawn Chances**. Also, mobs disappear from areas probably because the game or the mod are set to save energy and memory by removing mobs that are far from the player of a certain distance. They disappear even by restarting the world, so when you quit and reopen the world you find no animals where you expected them to be.
